### ISSUE TITLE
Description of System Technology

### ISSUE THREAD

**Issue by tvrjuana** [takchiks/ratemylab#53]
Describe the system technology and/or integration of the system.

**Comment by tvrjuana** [comment]
**System Technology**
A technological system is defined as a system that takes an input, changes it according to the system's use, and then produces an outcome. Therefore the system technology that was used in this project is comprised of the professor UI and what the professor will be inputting into it. For example, when the professor is first logging in, they will be prompted to change their given password, and then their new password will be stored in the database, (provided by the database team). The professor will then be able to choose which labs they would like to assign to their students. As a result, the students, by using the student UI, will be able to rate their assigned labs.

**Comment by tvrjuana** [comment]
**Integration of the System**
System integration is the process of connecting two or more systems together to exchange data. This can be achieved by direct database access, web services, or messaging. We achieved system integration through direct database access and web services.

We connected to the database via connect.php. There you will find the variable connections assigned to PDO (a lightweight, consistent interface for accessing databases in PHP). By using your localhost and the database name we made a connection from our UI to the database "RateMyLab_new".  When running PDO it will also require the database username and database password.
 
![connect](https://user-images.githubusercontent.com/47215199/101388373-daf58600-388d-11eb-9fe4-adf188d58c22.PNG)

To connect to the ratemylab.cs.gsu.edu/ providing by the university we had to upload our PHP files through FileZilla onto the server folder and to do this we had to connect to the VPN to gain access to the website.
